src/main/java/com/example/book/store/controller/OrderController.java

```diff
@@ -0,0 +1,72 @@
+package com.example.book.store.controller;
+
+import com.example.book.store.dto.order.OrderDto;
+import com.example.book.store.dto.order.OrderItemResponseDto;
+import com.example.book.store.dto.order.OrderRequestDto;
+import com.example.book.store.model.Order;
+import com.example.book.store.service.OrderService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "order management", description = "Endpoints for managing order")
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/api")
+public class OrderController {
+    private final OrderService orderService;
+
+    @Operation(
+            summary = "Place an order",
+            description = "Allows a user to place an order by sending a "
+                              + "POST request to /api/orders."
+    )
+    @PreAuthorize("hasRole('ROLE_USER')")
```

Comment:
this setup won't allow Admins to access endpoints annotated with ` @PreAuthorize("hasRole('ROLE_USER')")`. let's remove them. They are configured accessible only for authenticated users in Security Config anyway